src/EncompassRest/LoanPipeline/LoanPipelineCursor.cs

```diff
@@ -54,12 +54,7 @@ public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count, IEnumera
         {
             var retrievedData = await Client.Pipeline.ViewPipelineCursorInternalAsync(CursorId, null, itemFields ?? Fields, start, count, cancellationToken, nameof(GetItemAsync), async response =>
             {
-                var list = await response.Content.ReadAsAsync<List<LoanPipelineData>>().ConfigureAwait(false);
-                if (list.Count == 0)
-                {
-                    throw await RestException.CreateAsync("Failed to retrieve pipeline data", response).ConfigureAwait(false);
-                }
-                return list;
+                return await response.Content.ReadAsAsync<List<LoanPipelineData>>().ConfigureAwait(false);
```

Comment:
you can actually just return the `Task` and not return await it so it becomes the following.

`response => response.Content.ReadAsAsync<List<LoanPipelineData>>()`